### ISSUE TITLE
Enable DB data checksums by default?

### ISSUE THREAD

**Issue by mtbc** [ome/omero-install#181]
https://www.postgresql.org/docs/9.6/static/app-initdb.html mentions `--data-checksums` "to help detect corruption by the I/O system". Some searching online suggests negligible cost *if* combined with appropriate settings for basic options like `shared_buffers`. Perhaps we should be including it in our `initdb` workflows.

**Comment by joshmoore** [comment]
Perhaps something for https://github.com/openmicroscopy/ansible-role-postgresql ?

**Comment by mtbc** [comment]
True, could go into https://github.com/openmicroscopy/ansible-role-postgresql/blob/3.0.1/tasks/main.yml#L46. I also thought we could include it in our walkthroughs on https://docs.openmicroscopy.org/omero/5.4.7/sysadmins/unix/server-installation.html but now I'm surprised to find that I don't see where we tell people to use `initdb`.
